src/lib/services/transfer/__tests__/endpoint-manager/endpoint.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { createStorage } from '../../../../core/storage';
+import { endpointManager } from '../..';
+import { mirror } from '../../../../../__mocks__/handlers';
+
+const { endpoint } = endpointManager;
+
+const ENDPOINT = 'c591c905-2674-4227-9d31-1ff9485945a7';
+
+describe('endpoint', () => {
+  beforeEach(() => {
+    createStorage('memory');
+  });
```

Comment:
FWIW, you shouldn't actually need to do this any more (call `createStorage` in tests or runtime – #77).